game/src/trade/trade_menu.asm

```diff
@@ -0,0 +1,62 @@
+INCLUDE "game/src/common/constants.asm"
+
+SECTION "Load Parts Trade Menu", ROMX[$5a31], BANK[$7]
+LoadPartsTradeMenu:
+  ld b, 8
+  ld c, 2
+  ld e, $BA
+  call JumpLoadTilemap
+  ld a, [$C6F0]
+  ld c, a
+  ld a, [$C787]
+  or a
+  ret z
+
+.partA
+  and $7F
+  ld b, 0
+  call JumpTable_294
+  ld hl, $C6A2
```

Comment:
C6A2 should be cBUF01, which'll automatically take advantage of the extended buffer used in tr_EN.